docs/docsite/rst/dev_guide/testing/sanity/index.rst

```diff
@@ -27,7 +27,7 @@ For information on how to run these tests, see :ref:`sanity testing guide <testi
    no-dict-iterkeys
    no-dict-itervalues
    no-get-exception
-   no-illegal-filenames
+   no-illegal-file names
    no-main-display
```

Comment:
Sorry in this instance we can't change filenames to file names as it's the official name of a test. So please back out this change.